CS483/CS483/Kartaclysm/KartGame.cpp

```diff
@@ -28,6 +28,7 @@ bool Kartaclysm::KartGame::Init()
 	HeatStroke::KeyboardInputBuffer::CreateInstance(m_pWindow);
 	HeatStroke::JoystickInputBuffer::CreateInstance(m_pWindow);
 	HeatStroke::AudioPlayer::CreateInstance();
+	HeatStroke::AudioPlayer::Instance()->PreloadSoundEffects("CS483/CS483/Kartaclysm/Data/DevConfig/Preload.xml");
```

Comment:
Because music is streamed. Loading a music file is huge, so the correct practice is simply to stream it instead.